src/WooCommerceCheckout.php

```diff
@@ -118,4 +122,34 @@ public static function checkConfirmationEmailField() {
     }
   }
 
+  /**
+   * Check if we are using Amazon V2 in the checkout.
+   *
+   * @return bool
+   */
+  public static function isAmazonPayV2Checkout(): bool {
+    $is_amzon_pay_active = is_plugin_active('woocommerce-gateway-amazon-payments-advanced/woocommerce-gateway-amazon-payments-advanced.php') ?: FALSE;
+    if ($is_amzon_pay_active && isset(WC()->session)) {
+      if(version_compare(WC_AMAZON_PAY_VERSION, '2.0', '>=') && WC()->session->get('amazon_checkout_session_id') !== null) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  /**
+   * Remove Billing address when using Amazon Pay V2.
+   *
+   * * @return array
+   *   checkout fields.
+   * 
```

Comment:
Minor: We don't document the parameters and return value of a function if it is a defined hook that is already documented elsewhere (as indicated by `@implements`).
```suggestion
```
(Note that there was a duplicate asterisk before `@return`.)

Reply:
Noted. :)